react-ui-kit/src/CheckboxGroup/CheckboxGroupItem.ts

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+import { ICheckboxProps } from '../Checkbox';
+
+import { ITEM_MARGIN_BOTTOM_BIG, ITEM_MARGIN_BOTTOM_SMALL } from './constants';
+
+type CheckboxSize = ICheckboxProps['size'];
+
+export interface ICheckboxGroupItemProps {
+  size?: CheckboxSize;
```

Comment:
Может здесь указать просто:
```suggestion
  size?: ICheckboxProps['size'];
``` 
Чуть выше тоже такая штука есть

Reply:
Можно и так